### ISSUE TITLE
Default date for date filters does not get applied

### ISSUE THREAD

**Issue by LordRazen** [octobercms/october#5769]
October 3.5.12

I tried to activate the default date for date filters and I dont think it work anymore:

october/test/models/user/scopes.yaml
``` 
    created_at:
        label: Created
        type: date
        showWeekNumber: true
        default: '2020-01-02'
        # conditions:
        #     equals: true
        #     between: created_at >= ':after' AND created_at <= ':before'
```

https://docs.octobercms.com/3.x/element/filter/scope-date.html#php-interface
According to the documentation it should still work, but it doesnt. 

No date gets applied.

**Comment by daftspunk** [comment]
Hi @LordRazen 

This should be fixed in v3.6.0

Thanks!

**Comment by LordRazen** [comment]
Thanks for fixing
